tests/FSharp.AWS.DynamoDB.Tests/MetricsCollectorTests.fs

```diff
@@ -139,94 +137,94 @@ type ``Metrics Collector Tests`` (fixture : TableFixture) =
                     cu > 0
                 | _ -> false @>
 
-    let [<Fact>] ``Collect Metrics on Scan`` () =
+    let [<Fact>] ``Collect Metrics on BatchPutItem`` () =
         let hk = guid()
         let gsk = guid()
-        let items = [| for i in 0 .. 1000 -> mkItem hk gsk i |]
+        let items = [| for i in 0 .. 24 -> mkItem hk gsk i |]
         for item in items do
           table.PutItem item |> ignore
 
         let collector = TestCollector()
         let sut = table.WithMetricsCollector(collector.Collect)
 
-        let items = sut.Scan()
+        let _ = sut.BatchPutItems(items |> Seq.map (fun i -> { i with LocalAttribute = 1000 }))
 
         test <@ match collector.Metrics with
```

Comment:
In general I try to keep all assertion logic inlined, though this is on the unwieldy side here
Let me know if you want me to move this to a helper function and/or tidy it.